File: number_calculate.py
def find_max(a, b, c):
    return max(a, b, c)

def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(n**0.5)+1):
        if n % i == 0:
            return False
    return True

def even_numbers(n):
    for i in range(1, n+1):
        if i % 2 == 0:
            print(i, end=' ')

def factorial(n):
    result = 1
    for i in range(2, n+1):
        result *= i
    return result

File: number_calculate.java
public class MoreFunctions {
    public static int findMax(int a, int b, int c) {
        return Math.max(a, Math.max(b, c));
    }

    public static boolean isPrime(int n) {
        if (n < 2) return false;
        for (int i = 2; i <= Math.sqrt(n); i++) {
            if (n % i == 0) return false;
        }
        return true;
    }

    public static void evenNumbers(int n) {
        for (int i = 1; i <= n; i++) {
            if (i % 2 == 0) {
                System.out.print(i + " ");
            }
        }
        System.out.println();
    }

    public static int factorial(int n) {
        int result = 1;
        for (int i = 2; i <= n; i++) {
            result *= i;
        }
        return result;
    }

    public static void main(String[] args) {
        System.out.println(findMax(3, 7, 5));    // Output: 7
        System.out.println(isPrime(11));         // Output: true
        evenNumbers(10);                         // Output: 2 4 6 8 10
        System.out.println(factorial(5));        // Output: 120
    }
}
